parameter/TypeElement.cpp

```diff
@@ -76,8 +76,9 @@ bool CTypeElement::hasMappingData() const
 // Element properties
 void CTypeElement::showProperties(std::string& strResult) const
 {
-    (void)strResult;
-    // Prevent base from being called in that context!
+    showDescriptionProperty(strResult);
+    // Prevent base from being called from the Parameter Type context.
```

Comment:
Even if the description belongs to element, the field is appended in most cases to the parametertype and not to the parameter class itself. So, we had to forward only the retrieval of the description property field to minimize the changes. To avoid this, we shall have attached the description field to the parameter class properly, which might be more tricky.